apis/python/tests/test_tiledbsc.py

```diff
@@ -5,19 +5,32 @@
 import pytest
 import tempfile
 import os
+from pathlib import Path
 
-def test_import_anndata(request):
+HERE = Path(__file__).parent
+
+@pytest.fixture
+def h5ad_file(request):
     # Make sure this works regardless of from what directory level the `python -m pytest ...` is invoked
     ourdir = request.fspath.dirname
```

Comment:
looks like this `ourdir` isn't needed anymore -- looks like we can remove it

i took care to make sure people could do

* `python -m pytest tests` from `.../TileDB-SingleCell/apis/python`
* `python -m pytest apis/python/tests` from `.../TileDB-SingleCell`

i just verified this still works -- you're getting the same result from `Path(__file__)`

Reply:
Good call! I just pushed another commit to remove it.